alerting/src/main/kotlin/org/opensearch/alerting/transport/TransportIndexMonitorAction.kt

```diff
@@ -486,6 +486,13 @@ class TransportIndexMonitorAction @Inject constructor(
                 .setIfPrimaryTerm(request.primaryTerm)
                 .timeout(indexTimeout)
 
+            log.info(
```

Comment:
I would like to suggest put the monitor body into debug level log which is a long string